src/paredit/extension.ts

```diff
@@ -1,19 +1,19 @@
 'use strict';
-import { StatusBar } from './statusbar';
 import * as vscode from 'vscode';
 import {
   commands,
-  window,
+  ConfigurationChangeEvent,
   Event,
   EventEmitter,
   ExtensionContext,
+  window,
   workspace,
-  ConfigurationChangeEvent,
 } from 'vscode';
+import { EditableDocument } from '../cursor-doc/model';
 import * as paredit from '../cursor-doc/paredit';
 import * as docMirror from '../doc-mirror/index';
-import { EditableDocument } from '../cursor-doc/model';
 import { assertIsDefined } from '../utilities';
+import { StatusBar } from './statusbar';
```

Comment:
Do you have something that autocleans the imports? If so, please disable it on the Calva codebase. We welcome code quality updates in separate PRs (after a discussion, at least). When changes like these come together with some PR with some other topic, they slow the reviews down too much, imo. They also create potential merge conflicts of a slightly unnecessary nature.